src/entities/algo.go

```diff
@@ -9,6 +9,7 @@ const (
 
 	Babyjubjub Curve = "babyjubjub"
 	Secp256k1  Curve = "secp256k1"
+	X25519     Curve = "x25519"
```

Comment:
Since everything is a curve I didn't want to prefix it by that and checking around I see many references to `x25519` to the curve itself https://cryptography.io/en/latest/hazmat/primitives/asymmetric/x25519/ .

Even the documentation of the client who request it also refers to it that way
https://docs.waves.tech/en/blockchain/waves-protocol/cryptographic-practical-details#creating-a-private-key-from-a-seed